README.md

```diff
@@ -12,23 +12,46 @@ npm i -g @oat-sa/tao-extension-release
 
 ## Release an extension
 
-Run :
+Extension release now happens in 2 stages: the preparation and the release.
+
+### prepareRelease
+
+The first stage prepares the releasing branch, compiles assets and translations, and pushes the branch to the remote repo, ready to be tested, deployed or released.
 
 ```sh
-taoRelease
+taoRelease prepareRelease
 ```
 
-and follow the instructions
+### createRelease
 
-## Commandline arguments
+The second stage retrieves a prepared releasing branch from the remote repo, brings it up to date, and executes the release.
+
+```sh
+taoRelease createRelease
+```
 
-Commandline arguments:
+You will be prompted to follow the instructions.
 
- - `--base-branch` - branch to release from. 'develop' by default
- - `--branch-prefix` - releasing branch prefix. 'release' by default
- - `--origin` - git repository origin. 'origin' by default
- - `--release-branch` - branch to release to. 'master' by default
- - `--www-user` - www user. 'www-data' by default
+### oldWayRelease (deprecated)
+
+The `taoRelease oldWayRelease` command launches the sequence used by the old version (<= 0.4.1) of this tool. Functionally it is equivalent to running `prepareRelease` followed immediately by `createRelease`, but there is no opportunity to do any deployment or testing at the mid-point of the process.
+
+## Commandline arguments
+
+Commandline arguments to give you more control over the parameters of the release:
+
+| option | description | default | prepareRelease | createRelease | oldWayRelease |
```

Comment:
Even if it looks appealing I find difficult to find which option is available for my command.

Maybe separating it to 3 sections : 
 - General options (apply to all commands)
 - `prepareRelease` options 
 - `createRelease` options

Reply:
Done